packages/prepo-shared-contracts/test/SafeAccessControlEnumerable.test.ts

```diff
@@ -0,0 +1,211 @@
+  let owner: SignerWithAddress
+  let user1: SignerWithAddress
+  let user2: SignerWithAddress
+  let roleAMember: SignerWithAddress
+  let safeAccessControlEnumerable: SafeAccessControlEnumerable
+  const roleA = formatBytes32String('A')
+  const roleB = formatBytes32String('B')
+  const ZERO_HASH = formatBytes32String('')
+
+  const deploySafeAccessControlEnumerable = async (): Promise<void> => {
+    ;[deployer, user1, user2] = await ethers.getSigners()
+    owner = deployer
+    safeAccessControlEnumerable = await safeAccessControlEnumerableFixture()
+  }
+
+  const setupSafeAccessControlEnumerable = async (): Promise<void> => {
+    await deploySafeAccessControlEnumerable()
+    await safeAccessControlEnumerable.connect(deployer).grantRole(DEFAULT_ADMIN_ROLE, owner.address)
+    await safeAccessControlEnumerable.connect(owner).acceptRole(DEFAULT_ADMIN_ROLE)
+  }
+
+  describe('initial state', () => {
+    beforeEach(async () => {
+      await deploySafeAccessControlEnumerable()
+    })
+
+    it('sets default admin to deployer', async () => {
+      expect(await safeAccessControlEnumerable.hasRole(DEFAULT_ADMIN_ROLE, deployer.address)).to.eq(
+        true
+      )
+    })
+  })
+
+  describe('# setRoleAdminNominee', () => {
+    beforeEach(async () => {
+      roleAMember = user1
+      await setupSafeAccessControlEnumerable()
+      await safeAccessControlEnumerable.connect(owner).grantRole(roleA, roleAMember.address)
+      await safeAccessControlEnumerable.connect(roleAMember).acceptRole(roleA)
+      expect(await safeAccessControlEnumerable.hasRole(roleA, roleAMember.address)).to.eq(true)
+    })
+
+    it('reverts if not role admin and not role admin nominee', async () => {
+      const roleBAdmin = await safeAccessControlEnumerable.getRoleAdmin(roleB)
+      const roleBAdminNominee = await safeAccessControlEnumerable.getRoleAdminNominee(roleB)
+      expect(await safeAccessControlEnumerable.hasRole(roleBAdmin, user1.address)).to.eq(false)
+      expect(await safeAccessControlEnumerable.hasRole(roleBAdminNominee, user1.address)).to.eq(
+        false
+      )
+
+      await expect(
+        safeAccessControlEnumerable.connect(user1).setRoleAdminNominee(roleB, roleA)
+      ).to.be.revertedWith(
+        `AccessControl: account ${user1.address.toLowerCase()} is missing role ${DEFAULT_ADMIN_ROLE}`
+      )
+    })
+
+    it('reverts if role admin nominee but not role admin', async () => {
+      const roleBAdmin = await safeAccessControlEnumerable.getRoleAdmin(roleB)
+      await safeAccessControlEnumerable.setRoleAdminNominee(roleB, roleA)
+      const roleBAdminNominee = await safeAccessControlEnumerable.getRoleAdminNominee(roleB)
+      expect(roleBAdminNominee).to.eq(roleA)
+      expect(await safeAccessControlEnumerable.hasRole(roleBAdmin, roleAMember.address)).to.eq(
+        false
+      )
+      expect(
+        await safeAccessControlEnumerable.hasRole(roleBAdminNominee, roleAMember.address)
+      ).to.eq(true)
```

Comment:
```suggestion
      expect(
        await safeAccessControlEnumerable.hasRole(roleBAdminNominee, roleAMember.address)
      ).to.eq(true)
      expect(await safeAccessControlEnumerable.hasRole(roleBAdmin, roleAMember.address)).to.eq(
        false
      )
```
Same thing here as well, keep the ordering consistent.